### PR TITLE
add LINKs of FUNC

### LINKED ISSUE #189: README for FUGR
adopt according to #184

### PR DESCRIPTION
**Pull request by albertmink**
closes #189

### CODE REVIEW

File: file-formats/fugr/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ File | Cardinality | Definition | Schema | Example
 `<name>.fugr.sapl<name>.reps.json` | 1 | [`zif_aff_reps_v1.abap`](./type/zif_aff_reps_v1.intf.abap) |  [`reps.json`](./reps.json) | [`z_aff_example_fugr.fugr.saplz_aff_example_fugr.reps.json`](./examples/z_aff_example_fugr.fugr.saplz_aff_example_fugr.reps.json)
 `<name>.fugr.l<name>top.reps.abap` | 1 | ? | - | [`z_aff_example_fugr.fugr.lz_aff_example_fugrtop.reps.abap`](./examples/z_aff_example_fugr.fugr.lz_aff_example_fugrtop.reps.abap)
```
Comment (albertmink) [inline file-formats/fugr/README.md:8]:
```suggestion
`<name>.fugr.l<name>top.reps.abap` | 1 | [Function-Pool](https://help.sap.com/doc/abapdocu_751_index_htm/7.51/en-US/abapfunction-pool.htm) | - | [`z_aff_example_fugr.fugr.lz_aff_example_fugrtop.reps.abap`](./examples/z_aff_example_fugr.fugr.lz_aff_example_fugrtop.reps.abap)
```

**Review by crasovan — APPROVED** [review]
Looks good for me

**Review by crasovan — APPROVED** [review]
ok

File: file-formats/fugr/README.md [hunk 2]
```diff
@@ -3,11 +3,11 @@
 File | Cardinality | Definition | Schema | Example
 :--- | :---  | :--- | :--- | :---
 `<name>.fugr.json` | 1 | [`zif_aff_fugr_v1.abap`](./type/zif_aff_fugr_v1.intf.abap) | [`fugr.json`](./fugr.json) | [`z_example.fugr.json`](./examples/z_aff_example_fugr.fugr.json)
-`<name>.fugr.sapl<name>.reps.abap` | 1 | ? | - | [`z_aff_example_fugr.fugr.saplz_aff_example_fugr.reps.abap`](./examples/z_aff_example_fugr.fugr.saplz_aff_example_fugr.reps.abap)
+`<name>.fugr.sapl<name>.reps.abap` | 1 | [Function-Pool](https://help.sap.com/doc/abapdocu_751_index_htm/7.51/en-US/abapfunction-pool.htm) | - | [`z_aff_example_fugr.fugr.saplz_aff_example_fugr.reps.abap`](./examples/z_aff_example_fugr.fugr.saplz_aff_example_fugr.reps.abap)
```
Comment (huber-nicolas) [inline file-formats/fugr/README.md:6]:
```suggestion
`<name>.fugr.sapl<name>.reps.abap` | 1 | [Function-Pool](https://help.sap.com/doc/abapdocu_cp_index_htm/CLOUD/en-US/index.htm?file=abenfunction_pool_glosry.htm) | - | [`z_aff_example_fugr.fugr.saplz_aff_example_fugr.reps.abap`](./examples/z_aff_example_fugr.fugr.saplz_aff_example_fugr.reps.abap)
```

Comment (huber-nicolas) [reply file-formats/fugr/README.md:6]: Eveline and I think it would be better to always use the Cloud Abap Docu

Comment (albertmink) [reply file-formats/fugr/README.md:6]: sure, thanks

**Pull request merged**